### DISCUSSION TITLE
How to build a network when boundary conditions are input values

### DISCUSSION THREAD

**Discussion by Scienkiller** [lululxvi/deepxde#801] [Q&A]
Hello!
I am a student. Thanks for your excellent work, and I'm really interested in this. I saw the examples and had a question in my mind. In all cases, boundary conditions are given. We know that neural networks are used to solve equations, and the training data is obtained under certain boundary conditions. If the boundary conditions are also input values, such as inlet temperature and inlet velocity, what are the ideas for building a network?
In other words, when the boundary condition is also an input value, such as the value of the Dirichlet boundary condition, how should the boundary condition be set?

**Comment by lululxvi** [comment]
You can use DeepONet.
